src/matter/interaction/InteractionMessenger.ts

```diff
@@ -139,8 +139,18 @@ export class InteractionServerMessenger extends InteractionMessenger<MatterDevic
                         break;
                     case MessageType.InvokeCommandRequest:
                         const invokeRequest = TlvInvokeRequest.decode(message.payload);
-                        const invokeResponse = await handleInvokeRequest(invokeRequest, message);
-                        await this.exchange.send(MessageType.InvokeCommandResponse, TlvInvokeResponse.encode(invokeResponse));
+                        try {
+                            const invokeResponse = await handleInvokeRequest(invokeRequest, message);
+                            await this.exchange.send(MessageType.InvokeCommandResponse, TlvInvokeResponse.encode(invokeResponse));
+                        } catch (error) {
+                            if (error instanceof StatusResponseError) {
```

Comment:
I think this should be in the already existing catch statement below, nothing specific to InvokeCommandRequest

Reply:
Good Idea, adjusted